app/src/components/App/index.js

```diff
@@ -80,11 +79,6 @@ class App extends React.Component {
     console.log('App orderDetailAction called');
```

Comment:
This looks like it's basically a placeholder for a function that'll come later, but let's try and leave logs off of the master branch as much as possible.